[Docs migration] Cleans up investigation tools files - pt 2 (#565)
elastic/docs-content: solutions/security/investigate/open-manage-cases.md

    ::::{tip}

>     You can insert a Timeline link in the case description by clicking the Timeline icon (![Timeline icon](../../../images/security-add-timeline-button.png "title =20x20")).

Reviewer: same image bug here

* [Add and manage comments](/solutions/security/investigate/open-manage-cases.md#cases-manage-comments)

    ::::{tip}

>     Comments can contain Markdown. For syntax help, click the Markdown icon (![Click markdown icon](../../../images/security-markdown-icon.png "title =20x20")) in the bottom right of the comment.

Reviewer: image bug 😭

    ::::{important}
    Be mindful of the following:

>     * If the imported case had connectors attached to it, you’ll be prompted to re-authenticate the connectors. To do so, click **Go to connectors** on the **Import saved objects** flyout and complete the necessary steps. You can also access connectors from the **{{connectors-ui}}** page (find **{{connectors-ui}}** in the navigation menu or by using the [global search field](/explore-analyze/find-and-organize/find-apps-and-objects.md).

Reviewer: ```suggestion
    * If the imported case had connectors attached to it, you’ll be prompted to re-authenticate the connectors. To do so, click **Go to connectors** on the **Import saved objects** flyout and complete the necessary steps. You can also access connectors from the **{{connectors-ui}}** page (find **{{connectors-ui}}** in the navigation menu or by using the [global search field](/explore-analyze/find-and-organize/find-apps-and-objects.md)).
```

Author: Good catch!

>     * If the imported case had connectors attached to it, you’ll be prompted to re-authenticate the connectors. To do so, click **Go to connectors** on the **Import saved objects** flyout and complete the necessary steps. Alternatively, open the main menu, then go to **{{connectors-ui}}** (find **{{connectors-ui}}** in the navigation menu or by using the [global search field](/explore-analyze/find-and-organize/find-apps-and-objects.md))) to access connectors.

Author: ```suggestion
    * If the imported case had connectors attached to it, you’ll be prompted to re-authenticate the connectors. To do so, click **Go to connectors** on the **Import saved objects** flyout and complete the necessary steps. You can also access connectors from the **{{connectors-ui}}** page (find **{{connectors-ui}}** in the navigation menu or by using the [global search field](/explore-analyze/find-and-organize/find-apps-and-objects.md).
```
